[MAINT] Updated information about usage of UPDATE flag (#1558)
Haivision/srt: docs/API-functions.md

want to have some events reported as edge-triggered and others as
level-triggered, you can do two separate subscriptions for the same socket.

> **IMPORTANT**: The `srt_epoll_wait` function has no possibility to report
> `SRT_EPOLL_UPDATE` event. If you need an ability to get any possible flag,
> you need to use `srt_epoll_uwait`. Note that this function doesn't work with
> system file descriptors.

Reviewer: ```suggestion
**IMPORTANT**: The `srt_epoll_wait` function does not report
`SRT_EPOLL_UPDATE` events. If you need the ability to get any possible flag,
you must use `srt_epoll_uwait`. Note that this function doesn't work with
system file descriptors.
```

* `lwfds` and `lwnum`:A pointer and length of an array to write system sockets that are read-ready
* `lwfds` and `lwnum`:A pointer and length of an array to write system sockets that are write-ready

> Note that the flags are reported:

Reviewer: ```suggestion
Note that the following flags are reported:
```
